packages/office-ui-fabric-react/tslint.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": ["@uifabric/tslint-rules"],
   "rules": {
-    "deprecation": false,
+    "deprecation": true,
```

Comment:
You can just remove this line since the default is true

Reply:
Done!